_search-plugins/search-pipelines/ml-inference-search-response.md

```diff
@@ -390,12 +390,13 @@ The response confirms that the processor has generated text embeddings in the `p
 }
 ```
 
-### Example: GENAI use case
+### Example: Externally hosted model
```

Comment:
there is a Externally hosted model example at line 99 already. maybe we should come up with other title? @kolchfa-aws  https://github.com/opensearch-project/documentation-website/blob/faf070f13d52f6a56cb71ea31d05f9229044bc7a/_search-plugins/search-pipelines/ml-inference-search-response.md?plain=1#L99

Reply:
Updated